src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,63 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.25.0.dev2 (01/17/2020)
+------------------------
+
+New Features
+~~~~~~~~~~~~
+
+* Add Bandit Python linter (#8929)
+  `PR #8929 <https://github.com/pantsbuild/pants/pull/8929>`_
+
+* Tweak options parser to allow globs in filesystem specs (#8981)
+  `PR #8981 <https://github.com/pantsbuild/pants/pull/8981>`_
+
+* Introduce `SourcesSnapshot` for goals that only need source files to operate (#8980)
```

Comment:
Also mostly refactoring. This is only a new feature in terms of rule authors, which I don’t think we would put here because it’s a private API.